omod/src/main/webapp/resources/htmlforms/Covid19-DischargeForm.xml

```diff
@@ -0,0 +1,936 @@
+                                            </div>
+                                            <div class="col-md-2">
+                                                <label>Other therapy</label>
+                                                <obs conceptId="166445" answerConceptIds="90003,90004,90001" answerLabels="Yes, No, Unknown"
+                                                     style="radio" class="horizontal" id="other-therapy"/>
+                                            </div>
+                                            <div class="col-md-6">
+                                                <label>Specify Other therapy</label>
+                                                <obs id="specify-other-therapy" conceptId="166447" />
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-4">
+                                                <label>Oxygen therapy</label>
+                                                <obs id="oxygen-therapy" conceptId="166396" answerConceptIds="90003,90004,90001" answerLabels="Yes,No,Unknown"
+                                                     style="radio" class="horizontal"/>
+                                            </div>
+                                            <div class="col-md-4">
+                                                <label>O<sub>2</sub>Flow</label>
+                                                <div class="row">
+                                                    <obs id="02-flow" conceptId="166395" answerConceptIds="166392,166393,166394,166416,90001" answerLabel="1-5 L/min,6-10 L/min,11-15 L/min,>15,Unknown" style="checkbox" class="col-md-4"/>
+                                                </div>
+
+                                            </div>
+                                            <div class="col-md-4">
+                                                <label>Interface</label>
+                                                <div class="row">
+                                                    <obs id="interface" conceptId="166387" answerConceptIds="161253,166383,166384,166385,166386" answerLabel="Nasal prongs, Mask, Mask with reservoir, CPAP/NIV mask, HF nasal cannula"/>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+
+                            </div>
+                            <div class="tab-pane" id="supportive-care-section" >
```

Comment:
Use one card for all content in this tab

Reply:
Done